### PR TITLE
Additional Fixes

### PR DESCRIPTION
**Pull request by ronhowe**
Fixed image links.
Added Scariff icons.
Added Puck, Red 12 and Captain Khurgee.
Minor formatting.
Lore =  Blank. (standardization)

### CODE REVIEW

**Comment by ronhowe** [comment]
Any feedback?

**Review by thomasmarlin — CHANGES_REQUESTED** [review]
These changes are great!  Unfortunately we just merged another PR which created a conflict with this one.  The latest version now uses strings for the power/forfeit/ability/etc fields instead of numbers.  I'm guessing that's what the conflicts are about.  You should be able to pull the latest main branch into your local branch and hopefully resolving the conflicts will be easy.  

Post back here once those are resolved and I will get this merged.  These are all great updates. Thank you for taking time to work on this!

**Comment by ronhowe** [comment]
> These changes are great! Unfortunately we just merged another PR which created a conflict with this one. The latest version now uses strings for the power/forfeit/ability/etc fields instead of numbers. I'm guessing that's what the conflicts are about. You should be able to pull the latest main branch into your local branch and hopefully resolving the conflicts will be easy.
> 
> Post back here once those are resolved and I will get this merged. These are all great updates. Thank you for taking time to work on this!

Sounds good.  I'm on it.

**Comment by ronhowe** [comment]
All set.

File: Dark.json [hunk 1]
```diff
@@ -56940,6 +56939,31 @@
...
+      "id": 6823,
+      "side": "Dark",
+      "rarity": "U1",
+      "set": "Virtual Premium Set",
+      "front": {
+        "title": "•Captain Khurgee (V)",
+        "imageUrl": "https://res.starwarsccg.org/cards/Images-HT/starwars/VirtualPremium-Dark/large/captainkhurgee.gif?raw=true",
+        "type": "Character",
+        "subType": "Imperial",
+        "destiny": 2,
```
Comment (scottrick) [inline Dark.json:56953]: destiny/power/ability etc now should all be strings, not numbers.

File: Light.json [hunk 2]
```diff
@@ -24916,20 +24920,26 @@
...
         "title": "•Jabba's Prize/Jabba's Prize",
         "imageUrl": "https://res.starwarsccg.org/cards/Images-HT/starwars/Virtual0-Light/large/jabbasprizeback.gif?raw=true",
         "type": "Character",
         "subType": "Rebel",
-        "destiny": "7",
+        "destiny": "0",
         "power": "0",
         "ability": "0",
-        "deploy": "0",
+        "deploy": "0,
```
Comment (scottrick) [inline Light.json:24936]: This looks like invalid json.

File: Light.json [hunk 3]
```diff
@@ -58810,6 +58838,64 @@
...
+      "side": "Light",
+      "rarity": "U1",
+      "set": "Virtual Premium Set",
+      "front": {
+        "title": "•Red 12",
+        "imageUrl": "https://res.starwarsccg.org/cards/Images-HT/starwars/VirtualPremium-Light/large/red12.gif?raw=true",
+        "type": "Starship",
+        "subType": "Starfighter: X-Wing",
+        "uniqueness": "*",
+        "destiny": 2,
```
Comment (scottrick) [inline Light.json:58853]: Same as above, these should all be strings.

File: Light.json [hunk 4]
```diff
@@ -58810,6 +58838,64 @@
...
+      "side": "Light",
+      "rarity": "C2",
+      "set": "Virtual Premium Set",
+      "front": {
+        "title": "•Puck",
+        "imageUrl": "https://res.starwarsccg.org/cards/Images-HT/starwars/VirtualPremium-Light/large/puck.gif?raw=true",
+        "type": "Character",
+        "subType": "Rebel",
+        "destiny": null,
+        "power": 2,
```
Comment (scottrick) [inline Light.json:58887]: Same as above.

**Review by scottrick — CHANGES_REQUESTED** [review]

**Comment by ronhowe** [comment]
Yeah.  There was a lot of churn right when I submitted this.

I'll get this all cleane dup.

**Comment by ronhowe** [comment]
I'm going to close this PR so I can get it all in a good state.

**Pull request closed without merging**